src/arrays.js

```diff
@@ -5,7 +5,11 @@
  * @return {Boolean}
  */
 function sameFirst(a, b) {
-
+  if (a[0] ===  b[0]) {
+    return true;
```

Comment:
`return`ing true after checing that the statement `a[0] === b[0]` is redundant. You can save code and headaches down the road by just returning the conditional.